src/main/java/com/iridium/iridiumskyblock/listeners/PlayerJoinQuitListener.java

```diff
@@ -26,7 +26,7 @@ public void onJoin(PlayerJoinEvent event) {
                 PlayerUtils.sendBorder(player, island)
         );
 
-        if (player.isOp()) {
+        if (player.isOp() && IridiumSkyblock.getInstance().getConfiguration().patreonMessage) {
```

Comment:
This code can be simplified
```suggestion
        if (player.isOp() && true) {
```

Reply:
LOOOL